src/extension/report.ts

```diff
@@ -0,0 +1,50 @@
+import * as Sentry from "@sentry/node";
+import { v4 as uuidv4 } from "uuid";
+import {
+    workspace,
+} from "vscode";
+
+export class CrashReporter {
+    private enableCrashReporting: any;
+
+    constructor(
+        enableCrashReporting: boolean,
+    ) {
+        this.enableCrashReporting = enableCrashReporting;
+    }
+
+    public captureError(area: string, error: Error) {
```

Comment:
I think we can add a return type of `string | undefined` ?

Reply:
I thought about that too. Ok will change it.

Reply:
I returned an array of string instead, to cater for the sentryId and sentryPrompt.